portfolio/src/main/webapp/creative-works.html

```diff
@@ -47,7 +48,7 @@ <h1>Creative Works</h1>
             <img src="images/disposable.jpg" class="card-img-top"
                 alt="Disposable camera picture of a man holding a snake on the Brooklyn Bridge">
             <div class="card-body">
-              <p class="card-text">Disposable Diaries 1, August 2019</p>
+              <p class="card-text"><i>Disposable Diaries 1</i>, August 2019</p>
```

Comment:
For your personal site this is okay. in general, it would be better to not style explicitly in html (e.g. <i> tags).

I read this article recently about the two different philosophies of css frameworks and it might be helpful:
https://adamwathan.me/css-utility-classes-and-separation-of-concerns/#:~:text=One%20of%20the%20best%20practices,be%20made%20in%20your%20CSS.&text=See%20that%20.

Reply:
hey Ian! Is this wrt to the `<i>` tags specifically? Also, thank you for sharing that article - I was wondering about how to separate concerns with bootstrap's approach and it's cool to see the two different philosophies and an argument for utility classes :)